components/MovieList/index.js

```diff
@@ -0,0 +1,31 @@
+import useSWR from "swr";
+import { StyledDiv } from "./style";
+
+const fetcher = async (url) => {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data;
+};
+
+const MovieList = () => {
+  const API_KEY = process.env.NEXT_PUBLIC_MOVIE_HUB_API_KEY;
```

Comment:
I think this will not work. Please have a look at Frank's question in the Discussions-Board. It is complete the same thing.